packages/babel-plugin-transform-flow-strip-types/src/index.js

```diff
@@ -57,7 +57,14 @@ export default function ({ types: t }) {
           node = node.expression;
         } while (t.isTypeCastExpression(node));
         path.replaceWith(node);
-      }
+      },
+
+      ImportSpecifier(path) {
+        const { node } = path;
+        if (node.importKind !== null) {
```

Comment:
Sure I can do that. Update coming right up!

The only kinds are `"type"`, `"typeof"` and `null`. I don't expect we'll add a `"value"` since that's implied by `null` -- AKA leaving off any kind of keyword modifier.